From 6cc8c0184ba3b2c7aa6d89b2738959a98333d519 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Mon, 13 Aug 2012 19:43:18 +0200
Subject: [PATCH] sgxinit.c: fix crash for DM3730 (Overo Fire COMs)
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit


Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 services4/srvkm/devices/sgx/sgxinit.c |    4 ----
 1 files changed, 0 insertions(+), 4 deletions(-)

diff --git a/services4/srvkm/devices/sgx/sgxinit.c b/services4/srvkm/devices/sgx/sgxinit.c
index 052618d..c811758 100755
--- a/services4/srvkm/devices/sgx/sgxinit.c
+++ b/services4/srvkm/devices/sgx/sgxinit.c
@@ -409,10 +409,6 @@ PVRSRV_ERROR SGXInitialise(PVRSRV_SGXDEV_INFO	*psDevInfo,
         }
 
 #else
-        if(!(cpu_is_omap3530() || cpu_is_omap3517()))
-	{
-               OSWriteHWReg(psDevInfo->pvRegsBaseKM, 0xFF08, 0x80000000);//OCP Bypass mode
-	}
 #endif
 
 
-- 
1.7.6.5

